dotCMS/src/main/java/com/dotcms/rest/api/v1/page/ContainerView.java

```diff
@@ -0,0 +1,40 @@
+package com.dotcms.rest.api.v1.page;
+
+import com.dotmarketing.beans.ContainerStructure;
+import com.dotmarketing.portlets.containers.model.Container;
+
+import java.io.Serializable;
+import java.util.List;
+
+/**
+ * @author Will Ezell
+ * @author Jose Castro
+ * @version 4.2
+ * @since Oct 6, 2017
+ */
+public class ContainerView implements Serializable {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a "static final long serialVersionUID" field to this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2057)